tests/test_django/app/models.py

```
@@ -56,62 +93,75 @@ def age(self) -> int:
+    description = models.TextField[str, str]()
 
     @override
     def __str__(self) -> str:
         """Return the string representation of the author profile."""
-        return f"{self.author.name}: {self.description}"
+        author_name = self.author.name if self.author else "No author"
+        return f"{author_name}: {self.description}"
 
 
+@final
 class PublisherWebsite(BaseModel):
```

Ruff (0.14.10)

Model does not define `__str__` method

(DJ008)